dev/com.ibm.ws.concurrent.mp_fat_jakarta/fat/src/com/ibm/ws/concurrent/mp/fat/jakarta/MPContextProp1_3_EE10_Test.java

```diff
@@ -58,120 +66,120 @@ public static void tearDown() throws Exception {
+//     */
+//    @Test
+//    public void testCustomContextIsNotPropagatedByDefault() throws Exception {
+//        runTest(server, APP_NAME + "/MPContextProp1_3_EE10_TestServlet", testName);
+//    }
+//
+//    /**
+//     * Verify that custom thread context types are propagated if configured on the ContextServiceDefinition.
+//     */
+//    @Test
+//    public void testCustomEEContextIsPropagatedWhenConfigured() throws Exception {
+//        runTest(server, APP_NAME + "/MPContextProp1_3_EE10_TestServlet", testName);
+//    }
+//
+//    /**
+//     * Verify that custom thread context types are propagated if configured on the builder.
+//     */
+//    @Test
+//    public void testCustomMPContextIsPropagatedWhenConfigured() throws Exception {
+//        runTest(server, APP_NAME + "/MPContextProp1_3_EE10_TestServlet", testName);
+//    }
+//
+//    /**
+//     * java:comp/DefaultManagedExecutorService can be used interchangeably as a MicroProfile ManagedExecutor
+//     * or as a Jakarta EE ManagedExecutorService.
+//     */
+//    @Test
+//    public void testDefaultManagedExecutorService() throws Exception {
+//        runTest(server, APP_NAME + "/MPContextProp1_3_EE10_TestServlet", testName);
+//    }
+//
+//    /**
+//     * Jakarta EE Concurrency should not interfere with MicroProfile FaultTolerance Asynchronous
+//     * when only the MicroProfile annotation is present.
+//     */
+//    @Test
+//    public void testFaultToleranceAsyncMethod() throws Exception {
+//        runTest(server, APP_NAME + "/MPContextProp1_3_EE10_TestServlet", testName);
+//    }
+//
+//    /**
+//     * A method that is annotated with Jakarta EE Concurrency's Asynchronous must fail
+//     * if the class is annotated with MicroProfile Fault Tolerance Asynchronous.
+//     */
+//    @Test
+//    public void testFaultToleranceClassLevelAsynchronousCollidesWithMethod() throws Exception {
+//        runTest(server, APP_NAME + "/MPContextProp1_3_EE10_TestServlet", testName);
+//    }
+//
+//    /**
+//     * A method that is annotated with Jakarta EE Concurrency's Asynchronous must fail
+//     * if the method is also annotated with MicroProfile Fault Tolerance Asynchronous.
+//     */
+//    @Test
+//    public void testFaultToleranceCollidesOnSameAsyncMethod() throws Exception {
+//        runTest(server, APP_NAME + "/MPContextProp1_3_EE10_TestServlet", testName);
+//    }
+//
+//    /**
+//     * Use a MicroProfile ManagedExecutor and a Jakarta EE ManagedExecutorService
+//     * to create completion stages and combine the two to create dependent stages,
+//     * ensuring that each dependent stage runs with the configured context propagation
+//     * of the managed executor of the stage that creates the dependent stage.
+//     */
+//    @Test
+//    public void testIntermixMicroProfileAndJakartaEECompletionStages() throws Exception {
+//        runTest(server, APP_NAME + "/MPContextProp1_3_EE10_TestServlet", testName);
+//    }
```

Comment:
Don't leave commented out code/tests around with a good written explanation of why.  It will leave future developers wondering whether it really ought to be that way and whether tests ought to be switched back on.

Reply:
Sorry, that was an oversight, will remove.